server/db/migrate/20150123165155_create_deployments.rb

```diff
@@ -4,9 +4,21 @@ def change
       t.string :name
       t.integer :lifecycle_environment_id
       t.integer :organization_id
-      t.text :rhev_params
-      t.text :cfme_params
-      t.text :openstack_params
+      t.boolean :deploy_rhev
+      t.boolean :deploy_cfme
+      t.boolean :deploy_openstack
+      t.integer :rhev_hypervisor_host_id
+      t.integer :rhev_engine_host_id
```

Comment:
I assume these hosts should also have the appropriate relationships added to the model.  E.g. :has_one